.eslintrc.js

```diff
@@ -63,59 +63,57 @@ module.exports = {
                 ignoredNodes: ['JSXElement *']
             }
         ],
-        'key-spacing': [2, { beforeColon: false, afterColon: true }],
-        'keyword-spacing': [2, { before: true, after: true }],
-        'linebreak-style': 0,
-        'max-len': [2, 200, 2, { ignoreUrls: true, ignoreRegExpLiterals: true }],
-        'newline-after-var': 2,
-        'require-atomic-updates': 0,
-        'no-constant-condition': 2,
-        'no-dupe-class-members': 2,
-        'no-lonely-if': 2,
-        'no-multiple-empty-lines': [2, {
-            max: 1,
-            maxEOF: 0
-        }],
-        'no-prototype-builtins': 0,
-        'no-underscore-dangle': 2,
-        'no-var': 2,
-        'object-curly-spacing': [2, 'always'],
-        'object-curly-newline': [2, { consistent: true }],
-        'object-property-newline': [2, { allowAllPropertiesOnSameLine: true }],
-        'object-shorthand': [2, 'methods'],
-        'operator-linebreak': [0, 'before'],
-        'padded-blocks': [2, 'never'],
+        'key-spacing': ['error', { beforeColon: false, afterColon: true }],
+        'keyword-spacing': ['error', { before: true, after: true }],
+        'linebreak-style': ['error', 'unix'], // no carriage returns
+        'max-len': ['error', { code: 120 }], // be friendly to laptops
+        'newline-after-var': 'error',
+        'require-atomic-updates': 'warn', // okay to warn?
```

Comment:
reason this is not a warning is because of the ability to use fire and forget requests where you don't want it to properly await.


good example is for logging. if it's failing, a response isn't worth waiting for.